apps/lockfile-explorer/src/state/index.ts

```diff
@@ -0,0 +1,44 @@
+export enum ProjectType {
```

Comment:
In this repo, Microsoft generally adds a header banner to all source files (except for code samples/examples if I remember right):

```
// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
// See LICENSE in the project root for license information.

```

We should probably enable an ESLint rule to check that, because people forget it fairly often.

Reply:
Do you mean to add this to every file in the project?

Reply:
Yes. If it's a disruptive change, you could do it in a separate PR.

Reply:
added